dev/storage_base.h

```diff
@@ -885,15 +924,40 @@ namespace sqlite_orm {
                 return result;
             }
 
-            void drop_table_internal(sqlite3* db, const std::string& tableName) {
+            void drop_table_internal(sqlite3* db, const std::string& tableName, bool ifExists) {
                 std::stringstream ss;
-                ss << "DROP TABLE " << streaming_identifier(tableName) << std::flush;
+                ss << "DROP TABLE";
+                if(ifExists) {
+                    ss << " IF EXISTS";
```

Comment:
A trailing instead of a leading whitespace character is necessary.

Reply:
done